paper-typeahead-input.html

```diff
@@ -313,7 +340,7 @@
         }
       });
       var input = Polymer.dom(this.root).querySelector('paper-input');
-      input.$.input.autocomplete = 'address-level4';
+      input.$.input.autocomplete = this.autocomplete;
```

Comment:
Do you have any use case you think it's necessary to assign this autocomplete value of the input?

This line of code is to fix the Chrome Autofilling problem.